libs/vkd3d/command.c

```diff
@@ -8802,6 +8810,68 @@ static struct d3d12_command_list *unsafe_impl_from_ID3D12CommandList(ID3D12Comma
     return CONTAINING_RECORD(iface, struct d3d12_command_list, ID3D12GraphicsCommandList_iface);
 }
 
+static inline struct d3d12_command_list *d3d12_command_list_from_ID3D12GraphicsCommandListExt(ID3D12GraphicsCommandListExt *iface)
+{
+    struct d3d12_command_list_ext* list_ext = CONTAINING_RECORD(iface, struct d3d12_command_list_ext, ID3D12GraphicsCommandListExt_iface);
+    return CONTAINING_RECORD(list_ext, struct d3d12_command_list, command_list_ext);
+}
+
+static ULONG STDMETHODCALLTYPE d3d12_command_list_ext_AddRef(ID3D12GraphicsCommandListExt *iface)
+{
+    struct d3d12_command_list *command_list = d3d12_command_list_from_ID3D12GraphicsCommandListExt(iface);
```

Comment:
Seems a bit weird to chain these. Would be better I think to call d3d12_command_list_AddRef directly.